server/apps/main/helpers.py

```diff
@@ -657,4 +657,46 @@ def experience_titles_for_session(files):
     for f in files:
         if "uuid" in f['metadata'].keys():
             titles[f['metadata']['uuid']] = f['metadata']['description']
-    return titles
\ No newline at end of file
+    return titles
+
+def number_by_review_status(files):
+    """
+    Return a dictionary of review status: number for the My Stories page
+    """
+    list_status = ["approved", "not_reviewed", "rejected", "in_review"]
+    status = {s: 0 for s in list_status}
+
+    for f in files:
+        s = f['metadata']['data']['moderation_status']
+        status[s.replace(" ","_")] = status[s.replace(" ","_")] + 1
```

Comment:
I like the way you've managed to efficiently roll all four types into a single loop.